dll/win32/msvcrt/msvcrt.spec

```diff
@@ -218,10 +218,10 @@
 @ stub -version=0x600+ _CrtSetDbgBlockType
 @ stub -version=0x600+ _CrtSetDbgFlag
 @ stub -version=0x600+ _CrtSetDumpClient
-@ stub -version=0x600+ _CrtSetReportFile
+@ cdecl _CrtSetReportFile(long ptr)
```

Comment:
Is there `dbgonly` or something?

Reply:
https://github.com/reactos/reactos/pull/5663

Reply:
Is compatibility okay?

Reply:
Should we use the name of `MSVCRTD`?

Reply:
Some debug functions might be incompatible to original `msvcrt.dll`. Using such a debug function in `msvcrt.dll` instead of `msvcrtd.dll` will break compatibility.

Reply:
release versions should not have the debug asserts so there don't need to be control functions for them.

Reply:
The problem is that since we are coding the OS itself, and not a 3rd-party app and its separate redistributable CRT dll, the so-called "msvcrt.dll" system dll should be in its debug version when we compile ReactOS in Debug mode (which is what we do by default, and is being tested on the testbots, unless a Release version is compiled).

So, suppose we were to implement "Plan C)". Our inbox (possibly system) apps/modules would be linked against "msvcrtd.dll" ? But then they couldn't be run on Windows unless one copies also that dll. Doesn't sound good tbh.

"Plan B)" appears (to me at least) the best: the msvcrt.dll is the debug one in debug versions of the OS, and is the release one in the release version of ReactOS.